scss/_variables.scss

```diff
@@ -716,6 +716,7 @@ $card-border-radius:                $border-radius !default;
 $card-border-color:                 rgba($black, .125) !default;
 $card-inner-border-radius:          calc(#{$card-border-radius} - #{$card-border-width}) !default;
 $card-cap-bg:                       rgba($black, .03) !default;
+$card-cap-color:                    $gray-900 !default;
```

Comment:
Let's make this `$body-color` since that's what it was before.

Reply:
Done, thanks!